etcdserver/server.go

```diff
@@ -738,19 +742,18 @@ func (s *EtcdServer) run() {
 		case leases := <-expiredLeaseC:
 			s.goAttach(func() {
 				// Increases throughput of expired leases deletion process through parallelization
-				c := make(chan struct{}, maxPendingRevokes)
 				for _, lease := range leases {
 					select {
-					case c <- struct{}{}:
 					case <-s.stopping:
 						return
+					default:
 					}
+					revokerLimiter.Wait(context.Background())
 					lid := lease.ID
 					s.goAttach(func() {
```

Comment:
`maxPendingRevokes` is designed to take at most `maxPendingRevokes`(16 in this case) lease revoke at same time. By removing that, it means that there can be 1000 go routines that revoking the lease at the same time. Is that fine? I think the original design was to limit number of simultaneous revokes call to etcd.